src/main/resources/js/stores/PanelStore.js

```diff
@@ -0,0 +1,48 @@
+import alt from '../alt';
+
+import PanelConstants from '../constants/PanelConstants';
+
```

Comment:
Nit: lets do java style imports here, no spaces in between unless for good reason and alphabetical order